### ISSUE TITLE
Group view in the menu

### ISSUE THREAD

**Issue by ggabel** [monperrus/bibtexbrowser#77]
It would be helpful for our use case to have a list of the groups and subgroups of the bib-entries in the menu in the left side frame of the bibtexbrowser similar to jabref.
![groups_in_jabref_as_feature_request_in_bibtexbrowser](https://user-images.githubusercontent.com/32898122/31713004-0f9d27fa-b3fd-11e7-8c8c-e1e0f01c6144.png)

**Comment by monperrus** [comment]
you can view keywords as groups: all entries with a common keyword would be shown if you click on them (eg keyword "component" in the left frame of <http://bibtexbrowser.sourceforge.net/bibtexbrowser.php?frameset&bib=metrics.bib>)

for subgroups, howw ould you specify them in the bibtex file?

**Comment by ggabel** [comment]
I finally had the time to look into it. I found out the groups are done by JabRef with the comment-field, for the example above the code looks like this:
`@Comment{jabref-meta: grouping:
0 AllEntriesGroup:;
1 StaticGroup:GRUNDLAGEN\;0\;1\;\;\;\;;
1 StaticGroup:ANALYTIK & PRÄANALYTIK\;0\;1\;\;\;\;;
1 StaticGroup:PATHOBIOCHEMIE / DIAGNOSTIK / KLINIK\;0\;1\;\;\;\;;
2 StaticGroup:Fachübergreifendes\;0\;1\;\;\;\;;
2 StaticGroup:Endokrinologie & Stoffwechsel\;0\;1\;\;\;\;;
3 StaticGroup:Kohlenhydratstoffwechsel & Diabetes mell.\;0\;1\;\;\;\;;
3 StaticGroup:Lipidstoffwechsel\;0\;1\;\;\;\;;
3 StaticGroup:Schilddrüse & Nebenschilddrüse\;0\;1\;\;\;\;;
3 StaticGroup:Sonstige endokrine Systeme\;0\;1\;\;\;\;;
3 StaticGroup:Sonstige Stoffwechselprozesse\;0\;1\;\;\;\;;
2 StaticGroup:Gastroenterologie\;0\;1\;\;\;\;;
3 StaticGroup:Leber & Pankreas\;0\;1\;\;\;\;;
3 StaticGroup:Magen & Darmtrakt\;0\;1\;\;\;\;;
2 StaticGroup:Gynäkologie / Geburtshilfe\;0\;1\;\;\;\;;
2 StaticGroup:Hämatologie & hämatolog. Onkol.\;0\;1\;\;\;\;;
2 StaticGroup:Hämostaseologie\;0\;1\;\;\;\;;
2 StaticGroup:Immunologie / Autoimmunologie\;0\;1\;\;\;\;;
2 StaticGroup:Infektiologie / Sepsis\;0\;1\;\;\;\;;
2 StaticGroup:Kardiologie / Angiologie\;0\;1\;\;\;\;;
2 StaticGroup:Kinderheilkunde\;0\;1\;\;\;\;;
2 StaticGroup:Muskuloskelettales System\;0\;1\;\;\;\;;
2 StaticGroup:Nephrologie / Urologie / Elektrolyte / Säure-Basen-Haushalt\;0\;1\;\;\;\;;
2 StaticGroup:Neurologie / Psychiatrie\;0\;1\;\;\;\;;
2 StaticGroup:Onkologie\;0\;1\;\;\;\;;
2 StaticGroup:Pulmologie\;0\;1\;\;\;\;;
2 StaticGroup:Spurenelemente / Vitamine\;0\;1\;\;\;\;;
2 StaticGroup:Microbiome\;0\;1\;\;\;\;;
1 StaticGroup:ÜBERGEORDNETE ASPEKTE\;0\;1\;\;\;\;;
}`
Each entry has a group tag like this: 
`groups          = {Fachübergreifendes, Nephrologie / Urologie / Elektrolyte / Säure-Basen-Haushalt},`

**Comment by monperrus** [comment]
Do you mean that you would create the groups in Jabref, and then bibtexbrowser would interpret the grouping metadata from Jabref?

**Comment by ggabel** [comment]
Yes, that was my idea. For our page, I exported all the groups manually and used one bib-file per group. This feature would make it much easier for me but I don't know if there are many others that are interested in that and it justifies the effort.
